lib/TunnelCluster.js

```diff
@@ -103,7 +103,8 @@ module.exports = class TunnelCluster extends EventEmitter {
 
         remote.removeListener('close', remoteClose);
 
-        if (err.code !== 'ECONNREFUSED') {
+        if (err.code !== 'ECONNREFUSED'
+            && err.code !== 'ECONNRESET) {
```

Comment:
```suggestion
            && err.code !== 'ECONNRESET') {
```

There is a missing quote.